sqlmodel/main.py

```diff
@@ -387,8 +389,12 @@ def Field(
     sa_column_args: Union[Sequence[Any], UndefinedType] = Undefined,
     sa_column_kwargs: Union[Mapping[str, Any], UndefinedType] = Undefined,
     schema_extra: Optional[Dict[str, Any]] = None,
+    json_schema_extra: Optional[Dict[str, Any]] = None,
 ) -> Any:
-    current_schema_extra = schema_extra or {}
+    if json_schema_extra:
+        current_schema_extra = {"json_schema_extra": json_schema_extra}
+    else:
+        current_schema_extra = schema_extra or {}
```

Comment:
```suggestion
    current_schema_extra = schema_extra or {}
    if json_schema_extra:
        current_schema_extra["json_schema_extra"] = json_schema_extra
```

This is incorrect..
This will discard parameters specified via `schema_extra` if `json_schema_extra` is passed.
See explanation in the review